zql/docs/aggregate-functions/README.md

```diff
@@ -66,6 +70,49 @@ QUICKEST LONGEST     TYPICAL
 
 # Available Aggregate Functions
 
+## `and`
+
+|                           |                                                                |
+| ------------------------- | -------------------------------------------------------------- |
+| **Description**           | Returns the boolean value `true` if the provided expression evaluates to `true` for all input values. Contrast with [`or`](#or). |
+| **Syntax**                | `and(<expression>)`                                            |
+| **Required<br>arguments** | `<expression>`<br>A valid ZQL [expression](../expressions/README.md). |
+| **Optional<br>arguments** | None                                                           |
+| **Developer Docs**        | https://pkg.go.dev/github.com/brimsec/zq/reducer               |
+
+#### Example:
+
+Let's say you've been studying `weird` events and noticed that lots of
+connections have made one or more bad HTTP requests.
+
+```zq-command
+zq -f table 'count() by name | sort -r count' weird.log.gz
+```
+
+#### Output:
+```zq-output head:5
+NAME                                        COUNT
+bad_HTTP_request                            11777
+line_terminated_with_single_CR              11734
+unknown_HTTP_method                         140
+above_hole_data_without_any_acks            107
+...
+```
+
+To count the number of connections for which this was the _only_ category of
+`weird` event observed:
+
+```zq-command
+zq -f table 'only_bads=and(name="bad_HTTP_request") by uid | count() where only_bads=true' weird.log.gz
```

Comment:
nice query!

Reply:
is `where` already documented? I might be missing something but I didn't see it.

As a separate comment, this use of `where` could also be expressed with `filter` (unlike say `| .. c=count(), c2=count() where xyz | ...` which cannot).

Reply:
> is where already documented? I might be missing something but I didn't see it.

Answering myself: this is #1533

Reply:
@henridf: Indeed. I was planning on holding off merging _this_ PR until #1533 gets reviewed/merged because:

1. I reference `where` here (as you noticed), and,
2. There's formatting changes in #1533 that'll create merge conflicts, so I'll plan to resolve them in this branch before merging.

Your point about `filter` is a fine one. I could add the `c=count(),` and say in the description that we're including the total for comparison, which would cement the fact that `where` is _necessary_ rather than just convenient. However, I'm inclined to keep it simple as it is. The use of `where` in these simpler contexts reads slightly more intuitive to me than having separate pipeline elements for `filter` and `count()`, so I'm inclined to start referencing `where` so users become aware it's available.

Reply:
👍